### PR TITLE
[#10651] Broken Link on Instructor Help Page

### LINKED ISSUE #10651: Instructor help page: broken link
A user reported that this link is broken.
image

### PR DESCRIPTION
**Pull request by Jellwood**
Fixes #10651 

Once the links were found and the styling was determined to work out as mentioned in the discussion on the issue page, I started by using inline CSS, but I wanted to make sure that scaling worked in case the styling was supposed to scale. Because of this, I moved it to the CSS global styles page outside of this file. It should be able to scale if this needs to be propagated throughout. Photos below show the two affected links. I looked through all links on the page, and I believe I caught all of the modal window links.

StylizedFirstLink
StylizedSecondLink
FirstFixedScreenshot
SecondFixedScreenshot

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 1]
```diff
@@ -663,7 +665,7 @@ <h4 *ngIf="displaySubsection(0, 10)">Setting Up Questions</h4>
       </li>
     </ul>
     <p>
-      Technical details about how ranks are calculated are available <a (click)="openRankInfoModal(rankInfoHelp)" [tmRouterLink]="">here</a>.
+      Technical details about how ranks are calculated are available <span class=modal-window-class> <a (click)="openRankInfoModal(rankInfoHelp)" [tmRouterLink]="">here</a></span>.
```
Comment (moziliar) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:668]: No exactly sure how this part works without `ngTooltip` yet still showing the tool tip.

Comment (Jellwood) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html]: That was my bad, I had the local copy that made that and I was putting everything together in a segmented fashion and I thought I had pushed up the changes, I did not realize I made that mistake, I am so sorry. The new commit has the changes for the second modal link

Comment (moziliar) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html]: No worries. This is what code review is for. It is fine to have some git mistake.

**Comment by moziliar** [comment]
Just a question: how does this change prevent the user from right clicking -> opening in another tab which results in the "broken link" problem?
I am not sure if simply having a tooltip like this would resolve it entirely as the user might not know about the implication of the tooltip message.

**Comment by madanalogy** [comment]
I think the colour shouldn't be blue. Take a look at https://github.com/TEAMMATES/teammates/issues/10651#issuecomment-695215556 and https://github.com/TEAMMATES/teammates/issues/8465#issuecomment-741985931. @damithc any additional input for how the popup links are styled?

> I am not sure if simply having a tooltip like this would resolve it entirely as the user might not know about the implication of the tooltip message.

The idea from my understanding is to have a different styling so users don't immediately think it is a link.

**Comment by moziliar** [comment]
It is clickable after all. I think making it a black text with tooltip "click to open popup" might be an acceptable UX?

**Comment by madanalogy** [comment]
> It is clickable after all. I think making it a black text with tooltip "click to open popup" might be an acceptable UX?

Yes, based on the discussion in #10651 we want to go with black text with dashed underline

**Comment by Jellwood** [comment]
Alright I will work on getting that style working and cooperating with the rest of the page

**Comment by Jellwood** [comment]
I just pushed up commits that have the changes shown in the GIF, black text with dashed underline and no broken functionality, indicated in the GIF
TeammatesWorkingModalLinks

**Review by moziliar — APPROVED** [review]
LGTM. Changes are as expected!

**Review by madanalogy — APPROVED** [review]
LGTM

**Pull request merged**